### PR TITLE
Knocked-over chairs have materials.

### PR DESCRIPTION
**Pull request by harmonyn**
Fixes https://github.com/tgstation/tgstation/issues/28442.

### CODE REVIEW

File: code/game/objects/structures/beds_chairs/chair.dm [hunk 1]
```diff
@@ -234,6 +234,7 @@
 	throw_range = 3
 	hitsound = 'sound/items/trayhit1.ogg'
 	hit_reaction_chance = 50
+	materials = list(MAT_METAL = 500)
```
Comment (ChangelingRain) [inline code/game/objects/structures/beds_chairs/chair.dm]: but chairs are made of 2000 metal? knocking a chair over doesn't reduce the amount of metal in it.

Comment (bgobandit) [inline code/game/objects/structures/beds_chairs/chair.dm]: I was wondering how much fucking metal chairs were made of

Comment (DoctoreMad) [inline code/game/objects/structures/beds_chairs/chair.dm]: Isn't 2000 one/less than one sheet?

Comment (octareenroon91) [inline code/game/objects/structures/beds_chairs/chair.dm]: A chair costs one sheet. The var on metal sheets reads as follows:
`materials = list(MAT_METAL=MINERAL_MATERIAL_AMOUNT)`

File: code/game/objects/structures/beds_chairs/chair.dm [hunk 2]
```diff
@@ -317,6 +318,7 @@
 	max_integrity = 70
 	hitsound = 'sound/weapons/genhit1.ogg'
 	origin_type = /obj/structure/chair/wood
+	materials = null
```
Comment (KorPhaeron) [inline code/game/objects/structures/beds_chairs/chair.dm]: Shouldn't it be made of wood?

**Pull request merged**